CHANGELOG.md

```diff
@@ -9,14 +9,18 @@ All notable changes to this project will be documented in this file.
 - Reconciliation errors are now reported as Kubernetes events ([#178]).
 - Use cli argument `watch-namespace` / env var `WATCH_NAMESPACE` to specify
   a single namespace to watch ([#183]).
+- Local backend storage (deep-storage) has been replaced with HDFS-storage ([#187]).
```

Comment:
This is breaking (for the CRD at least)?

Reply:
marked as such in changelog - done